app.js

```diff
@@ -1,9 +1,14 @@
-const path = require("path");
+const fs = require('fs');
+const path = require('path');
 const express = require("express");
 const bodyParser = require("body-parser");
-
+const mongoose = require('mongoose');
 const youtubeRoutes = require("./routes/youtube");
 
+const username = encodeURIComponent("maestrodb");
```

Comment:
Let's try to use the best practice by storing info like creds in .env files. For now, it's fine but something to be updated on the next branches.